packages/tupaia-web/src/features/Visuals/View/View.tsx

```diff
@@ -13,7 +13,7 @@ import { formatDataValueByType } from '@tupaia/utils';
 import { MultiValueRow } from './MultiValueRow';
 import { DataDownload } from './DataDownload';
 import { DownloadFiles } from './DownloadFiles';
-
+import { DashboardInfoHover } from '../../DashboardItem/DashboardInfoHover';
```

Comment:
I would add `DashboardInfoHover` as an export to `DashboardItem/index.ts` and then this import can become `...... from `../../DashboardItem`